tests/foreman/ui/test_ansible.py

```diff
@@ -41,13 +41,15 @@ def test_positive_create_and_delete_variable(target_sat):
     :expectedresults: The variable is successfully created and deleted.
     """
     key = gen_string('alpha')
-    role = 'redhat.satellite.activation_keys'
+
+    SELECTED_ROLE = 'redhat.satellite.activation_keys'
+    proxy_id = target_sat.nailgun_smart_proxy.id
+    target_sat.api.AnsibleRoles().sync(data={'proxy_id': proxy_id, 'role_names': [SELECTED_ROLE]})
```

Comment:
@adarshdubey-star Okay, so as per updated description, we were testing importing roles with UI and had a locator issue so we moved to API for importing roles, so I think this isn't the way how it should be fixed, and we should test roles and variable together, but for this ansible variable tests this will be considered as part of setup, so I don't think to block the PR merge on this.